Python/initconfig.c

```diff
@@ -187,67 +187,41 @@ arg ...: arguments passed to program in sys.argv[1:]\n\
-         os.process_cpu_count(), and multiprocessing.cpu_count().  This can\n\
-         help users who need to limit resources in a container.\n\
--X dev : enable CPython's \"development mode\", introducing additional runtime\n\
-         checks which are too expensive to be enabled by default.  Effect of\n\
-         the developer mode:\n\
-          * Add default warning filter, as -W default\n\
-          * Install debug hooks on memory allocators: see the\n\
-            PyMem_SetupDebugHooks() C function\n\
-          * Enable the faulthandler module to dump the Python traceback on\n\
-            a crash\n\
-          * Enable asyncio debug mode\n\
-          * Set the dev_mode attribute of sys.flags to True\n\
-          * io.IOBase destructor logs close() exceptions\n\
--X faulthandler: enable faulthandler\n\
+-X cpu_count=N: override the return value of os.cpu_count();\n\
+         -X cpu_count=default cancels overriding; also PYTHON_CPU_COUNT\n\
+-X dev : enable Python development mode; also PYTHONDEVMODE\n\
+-X faulthandler: dump the Python traceback on fatal errors;\n\
+         also PYTHONFAULTHANDLER\n\
 -X frozen_modules=[on|off]: whether or not frozen modules should be used.\n\
-         The default is \"on\" (or \"off\" if you are running a local build).\n\
--X importtime: show how long each import takes.  It shows module name,\n\
-         cumulative time (including nested imports) and self time (excluding\n\
-         nested imports).  Note that its output may be broken in\n\
-         multi-threaded application.\n\
-         Typical usage is python3 -X importtime -c 'import asyncio'\n\
--X int_max_str_digits=number: limit the size of int<->str conversions.\n\
-         This helps avoid denial of service attacks when parsing untrusted\n\
-         data.  The default is sys.int_info.default_max_str_digits.\n\
-         0 disables.\n\
--X no_debug_ranges: disable the inclusion of the tables mapping extra location\n\
-         information (end line, start column offset and end column offset) to\n\
-         every instruction in code objects.  This is useful when smaller code\n\
-         objects and pyc files are desired as well as suppressing the extra\n\
-         visual location indicators when the interpreter displays tracebacks.\n\
--X perf: activate support for the Linux \"perf\" profiler by activating the\n\
-         \"perf\" trampoline.  When this option is activated, the Linux \"perf\"\n\
-         profiler will be able to report Python calls.  This option is only\n\
-         available on some platforms and will do nothing if is not supported\n\
-         on the current system.  The default value is \"off\".\n\
+         The default is \"on\" for installed Python and \"off\" for a local build.\n\
+         Also PYTHON_FROZEN_MODULES.\n\
+-X importtime: show how long each import takes; also PYTHONPROFILEIMPORTTIME\n\
+-X int_max_str_digits=number: limit the size of int<->str conversions;\n\
```

Comment:
`=number` or `=N`? There's a few `=N` in other places

```suggestion
-X int_max_str_digits=N: limit the size of int<->str conversions;\n\
```